extension/js/common/core/mime.ts

```diff
@@ -197,26 +197,40 @@ export class Mime {
 
   public static encode = async (body: SendableMsgBody, headers: RichHeaders, attachments: Attachment[] = [], type?: MimeEncodeType): Promise<string> => {
```

Comment:
I'm hesitant to merge this type of change if it ends up being used for regular pgp/mime messages. If we only used it for drafts, that would be ok, but I'd prefer not to alter with outgoing messages unless we have to.

Is there some benefit to wrapping outgoing messages this way that I overlooked?

Reply:
No, there is no immediate benefit. This was done as part of experiment, so I wanted to save this code.

Reply:
Got it, I'll leave this open